keyard/tests/test_integration_store.py

```diff
@@ -2,15 +2,33 @@
 import pytest
 
 from keyard import store
+from keyard import testutils
 
 
-@pytest.mark.xfail
+@pytest.mark.skipif(not testutils.etcd_is_available(), reason="etcd is missing")
```

Comment:
Your intention with this integration tests, is to integrate ETCD too? or just keyard logic?

I think you could build a ETCD mock without problems, looks really simple... than you dont need this skipif